TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/TestTele.java

```diff
@@ -70,30 +69,41 @@ public void runOpMode() {
             //OPERATOR FUNCTIONS
 
             //toggles dropper, make code that goes down and then back up later
-            if (controller2.a == Btn.PRESSING) {
-                robot.dropGameElement();
+            if (controller2.a == Btn.PRESSED) {
+                //robot.dropGameElement();
+                robot.jankToggleDropper();
             }
 
             //intake
-            //run intake based on how strong trigger pressed (i think)
-            boolean triggerPressed = Math.abs(controller2.right_trigger - Controller.TRIGGER_PRESSED) < .01;
-            if (triggerPressed) {
-                robot.setIntakePower(0.9 * controller2.right_trigger);
-            }
+            //run intake based on how strong the right trigger is pressed
+                robot.setIntakePower(0.9 * controller2.right_stick_y);
```

Comment:
get rid of this, implement toggle elsewhere (in Robot or DriveBase or something)